packages/falso/src/lib/programming-languages.ts

```diff
@@ -0,0 +1,35 @@
+import { rand } from './core';
+
+export function programmingLanguages() {
```

Comment:
I would suggest to rename this to singular
```suggestion
export function programmingLanguage() {
```